browser/src/VariantList/VariantFlag.tsx

```diff
@@ -2,7 +2,15 @@ import React from 'react'
 
 import { Badge } from '@gnomad/ui'
 
-export const FLAGS_CONFIG = {
+type Flag = {
+  [key: string]: {
```

Comment:
one small note here, I would rewrite this so `Flag` refers specifically to the `{label, level, formatTooltip}` bit. In other words, something like:

```
type Flag = {
  label: string
  level: 'info' | 'warning' | 'error' | 'success' | undefined
  formatTooltip: (input: any) => string
}

export const FLAGS_CONFIG: Record<string, Flag> = {
...
```

Reply:
Aha! Very nice, will do. Thanks for the suggestion and the code as example! I wasn't aware of the Record in TypeScript, I'll keep Record with generic typing in mind.